src/util/Injected.js

```diff
@@ -152,7 +155,6 @@ exports.LoadUtils = () => {
         });
 
         const message = {
-            ...options,
```

Comment:
I really want the ...options to overwrite all other options. But this can break something, we must be careful